src/components/VmDetail/index.js

```diff
@@ -70,6 +76,9 @@ class VmDetail extends Component {
     const icon = icons.get(iconId)
     const disks = vm.get('disks')
 
+    const onToggleRenderDisks = () => { this.setState({ renderDisks: !this.state.renderDisks }) }
```

Comment:
@mareklibra I agree using redux to store a form-like state (e.g. confirmation of a button and such) doesn't really make much sense. So in this case I'm also okay with leaving this as a stateful component as long as the state will be isolated in the smallest possible component - e.g.  the mentioned `ExpandableVmDisks`.